packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -162,8 +162,9 @@ export default class DocumentCollection {
     const indexId = options.indexId || (await this.getIndexId(indexFields))
     const { fields, skip = 0, limit = FETCH_LIMIT } = options
     // Mango wants an array of single-property-objects...
+    const sortOrder = options.sort ? Object.values(options.sort)[0] : 'desc'
```

Comment:
Where does the [0] come from ? The order of the values is not guaranteed so this may sort in 1 direction sometime and in another another time, right ?

I think we should just throw in case of double sort by the user.